x/ccv/provider/keeper/distribution.go

```diff
@@ -121,19 +117,36 @@ func (k Keeper) AllocateTokens(ctx sdk.Context) {
 			)
 			continue
 		}
+
+		// compute rewards for validators
+		consumerRewards := alloc.Rewards
 		voteMultiplier := math.LegacyOneDec().Sub(communityTax)
-		feeMultiplier := rewardsCollectedDec.MulDecTruncate(voteMultiplier)
+		validatorRewards := consumerRewards.MulDecTruncate(voteMultiplier)
+
+		// compute remaining rewards for the community pool
+		remaining := consumerRewards.Sub(validatorRewards)
+
+		// transfer validators rewards to distribution module account
+		validatorRewardsTrunc, validatorRewardsChange := validatorRewards.TruncateDecimal()
+		err = k.bankKeeper.SendCoinsFromModuleToModule(ctx, types.ConsumerRewardsPool, distrtypes.ModuleName, validatorRewardsTrunc)
```

Comment:
nit: s/`validatorRewards`/`validatorsRewards` to be consistent with comment above `// transfer validators rewards ...`